common/datavalues/src/vec.rs

```diff
@@ -265,11 +277,14 @@ impl<T> AlignedVec<T> {
         self.inner.extend(iter);
         let added = self.len() - len_before;
 
-        assert_eq!(added, cap)
+        if added != cap {
+            eprintln!("size hint was incorrect, this is UB. aborting");
+            std::process::abort()
```

Comment:
Is that good way to just abort?
Or we can use debug_assert! .

Reply:
> Is that good way to just abort?
> Or we can use debug_assert! .

Just follow the polars way. I will look at the debug_assert! .

Reply:
ok